js/viz/series/base_series.js

```diff
@@ -485,7 +485,35 @@ Series.prototype = {
         }
     },
 
-    _drawElements: function(animationEnabled, firstDrawing, translateAllPoints) {
+    _preparePositionSegments() {
+        const that = this;
+        const points = that._points || [];
+        const closeSegment = points[0] && points[0].hasValue() && that._options.closed;
```

Comment:
```suggestion
        const isCloseSegment = points[0]?.hasValue() && that._options.closed;
```
or even `isClosedSegment`?

To be honest, I'm not sure that we're checking segments here. Are u sure this name is most successful?